pkg/apis/ceph.rook.io/v1/resources.go

```diff
@@ -43,6 +43,8 @@ const (
 	ResourcesKeyFilesystemMirror = "fsmirror"
 	// ResourcesKeyCleanup represents the name of resource in the CR for the cleanup
 	ResourcesKeyCleanup = "cleanup"
+	// ResourcesKeyCleanup represents the name of resource in the CR for ceph-exporter
+	ResourcesKeyCephExporter = "ceph-exporter"
```

Comment:
No need for the ceph prefix, it keeps things simpler
```suggestion
	ResourcesKeyCephExporter = "exporter"
```